### PR TITLE
Update self hosted runner for an org steps

### LINKED ISSUE #6160: Self hosted runners missing step
### What article on docs.github.com is affected?

https://docs.github.com/en/actions/hosting-your-own-runners/adding-self-hosted-runners

### What part(s) of the article would you like to see updated?

In the organization settings gh runner config have been moved to their own page and the docs existing do not reference the additional step needed. 

The documentation should be updated to reflect the additional required step.

### Additional information

Docs:
image

Current Settings Page Sidebar:
image

### PR DESCRIPTION
**Pull request by 721p**
### Why:

**Closes #6160**

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [x] I have added a description and/or a video demo of the changes below (eg. a "before and after video")

### CODE REVIEW

**Comment by martin389** [comment]
### Summary
Closes https://github.com/github/docs/issues/6160

### Changes in this PR
| Article in staging  | Changes |
| --- | --- | 
“[Adding a self-hosted runner to an organization](https://docs-6161--patch-1.herokuapp.com/en/actions/hosting-your-own-runners/adding-self-hosted-runners#adding-a-self-hosted-runner-to-an-organization)” |  Updated steps in procedure.

### Versioning
dotcom

### Tests
All tests are passing! :green_apple:

File: data/reusables/github-actions/settings-sidebar-actions-runners.md [hunk 1]
```diff
@@ -0,0 +1 @@
+1. In the left sidebar, click **Actions**{% if currentVersion == "free-pro-team@latest" %}, then **Runners**{% endif %}.
```
Comment (lecoursen) [inline data/reusables/github-actions/settings-sidebar-actions-runners.md:1]:
```suggestion
1. In the left sidebar, click **Actions**{% if currentVersion == "free-pro-team@latest" %}, then click **Runners**{% endif %}.
```

File: data/reusables/github-actions/self-hosted-runner-navigate-to-repo-org-enterprise.md [hunk 2]
```diff
@@ -6,7 +6,5 @@
      1. In the left sidebar, click **Enterprise overview**. 
      1. {% endif %} In the enterprise sidebar, {% octicon "law" aria-label="The law icon" %} **Policies**.
 1. Navigate to the {% data variables.product.prodname_actions %} settings:
-   * **In an organization or repository**: click **Actions** in the left sidebar.
-     
-     ![Actions setting](/assets/images/help/settings/settings-sidebar-actions.png)
-   * {% if currentVersion == "free-pro-team@latest" %}**If using an enterprise account**{% elsif enterpriseServerVersions contains currentVersion and currentVersion ver_gt "enterprise-server@2.21" or currentVersion == "github-ae@latest" %}**If using an enterprise-level runner**{% endif %}: click **Actions** under "{% octicon "law" aria-label="The law icon" %} Policies".
+   * **In an organization or repository**: click **Actions** in the left sidebar{% if currentVersion == "free-pro-team@latest" %}, then **Runners**{% endif %}.
```
Comment (lecoursen) [inline data/reusables/github-actions/self-hosted-runner-navigate-to-repo-org-enterprise.md:9]:
```suggestion
   * **In an organization or repository**: click **Actions** in the left sidebar{% if currentVersion == "free-pro-team@latest" %}, then click **Runners**{% endif %}.
```

File: data/reusables/github-actions/self-hosted-runner-navigate-to-repo-org-enterprise.md [hunk 3]
```diff
@@ -6,7 +6,5 @@
...
+   * {% if currentVersion == "free-pro-team@latest" %}**If using an enterprise account**{% elsif enterpriseServerVersions contains currentVersion and currentVersion ver_gt "enterprise-server@2.21" or currentVersion == "github-ae@latest" %}**If using an enterprise-level runner**:{% endif %} Click **Actions** under "{% octicon "law" aria-label="The law icon" %} Policies".
```
Comment (lecoursen) [inline data/reusables/github-actions/self-hosted-runner-navigate-to-repo-org-enterprise.md:10]: This was missing in [this procedures](https://docs-6161--patch-1.herokuapp.com/en/actions/hosting-your-own-runners/using-labels-with-self-hosted-runners#creating-a-custom-label) but you should probably check it's not creating duplicate steps in any other procedures.

```suggestion
   * {% if currentVersion == "free-pro-team@latest" %}**If using an enterprise account**{% elsif enterpriseServerVersions contains currentVersion and currentVersion ver_gt "enterprise-server@2.21" or currentVersion == "github-ae@latest" %}**If using an enterprise-level runner**:{% endif %} Click **Actions** under "{% octicon "law" aria-label="The law icon" %} Policies", then click the **Runners** tab.
```

File: data/reusables/github-actions/self-hosted-runner-removing-a-runner.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-1. Under **Self-hosted runners**, locate the runner in the list. If your runner is in a group, click {% octicon "chevron-down" aria-label="The downwards chevron" %} to expand the list.
+1. Under {% if currentVersion == "free-pro-team@latest" %}_Runners_{% else %}_Self-hosted runners_{% endif %}, locate the runner in the list. If your runner is in a group, click {% octicon "chevron-down" aria-label="The downwards chevron" %} to expand the list.
```
Comment (lecoursen) [inline data/reusables/github-actions/self-hosted-runner-removing-a-runner.md:1]: I think we also need to use the new `settings-sidebar-actions-runners` reusable in the "Removing..." procedures or the new step to click "Runners" is missing.

File: content/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups.md [hunk 5]
```diff
@@ -104,7 +104,7 @@ You can update the access policy of a runner group, or rename a runner group.
```
Comment (lecoursen) [inline content/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups.md:104]: I think we need to also update `{% data reusables.github-actions.self-hosted-runner-configure-runner-group-access %}` above (in "Changing the access policy of a self-hosted runner group")?

**Review by lecoursen — COMMENTED** [review]
Thanks! @martin389, I _think_ I found a few more places to update, if you don't mind.

File: content/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups.md (hunk 5 above)
Comment (martin389) [reply content/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups.md:104]: Ooh yes, thanks for spotting that one! 🚀

File: data/reusables/github-actions/self-hosted-runner-removing-a-runner.md (hunk 4 above)
Comment (martin389) [reply data/reusables/github-actions/self-hosted-runner-removing-a-runner.md:1]: Fixed, thank you!

**Comment by martin389** [comment]
Thanks for the review @lecoursen ⚡ I've added in the fixes now

**Comment by Grcy3g8** [comment]
Thanks

File: data/reusables/github-actions/self-hosted-runner-navigate-to-repo-org-enterprise.md [hunk 6]
```diff
@@ -6,7 +6,5 @@
...
-   * **In an organization or repository**: click **Actions** in the left sidebar.
-     
-     ![Actions setting](/assets/images/help/settings/settings-sidebar-actions.png)
-   * {% if currentVersion == "free-pro-team@latest" %}**If using an enterprise account**{% elsif enterpriseServerVersions contains currentVersion and currentVersion ver_gt "enterprise-server@2.21" or currentVersion == "github-ae@latest" %}**If using an enterprise-level runner**{% endif %}: click **Actions** under "{% octicon "law" aria-label="The law icon" %} Policies".
+   * **In an organization or repository**: click **Actions** in the left sidebar{% if currentVersion == "free-pro-team@latest" %}, then click **Runners**{% endif %}.
+   * {% if currentVersion == "free-pro-team@latest" %}**If using an enterprise account**{% elsif enterpriseServerVersions contains currentVersion and currentVersion ver_gt "enterprise-server@2.21" or currentVersion == "github-ae@latest" %}**If using an enterprise-level runner**:{% endif %} Click **Actions** under "{% octicon "law" aria-label="The law icon" %} Policies"{% if currentVersion == "free-pro-team@latest" %}, then click the **Runners** tab{% endif %}.
```
Comment (lecoursen) [inline data/reusables/github-actions/self-hosted-runner-navigate-to-repo-org-enterprise.md:10]:
```suggestion
   * {% if currentVersion == "free-pro-team@latest" %}**If using an enterprise account**:{% elsif enterpriseServerVersions contains currentVersion and currentVersion ver_gt "enterprise-server@2.21" or currentVersion == "github-ae@latest" %}**If using an enterprise-level runner**:{% endif %} Click **Actions** under "{% octicon "law" aria-label="The law icon" %} Policies"{% if currentVersion == "free-pro-team@latest" %}, then click the **Runners** tab{% endif %}.
```

File: data/reusables/github-actions/self-hosted-runner-navigate-to-repo-org-enterprise.md [hunk 7]
```diff
@@ -6,7 +6,5 @@
      1. In the left sidebar, click **Enterprise overview**. 
      1. {% endif %} In the enterprise sidebar, {% octicon "law" aria-label="The law icon" %} **Policies**.
 1. Navigate to the {% data variables.product.prodname_actions %} settings:
-   * **In an organization or repository**: click **Actions** in the left sidebar.
-     
-     ![Actions setting](/assets/images/help/settings/settings-sidebar-actions.png)
-   * {% if currentVersion == "free-pro-team@latest" %}**If using an enterprise account**{% elsif enterpriseServerVersions contains currentVersion and currentVersion ver_gt "enterprise-server@2.21" or currentVersion == "github-ae@latest" %}**If using an enterprise-level runner**{% endif %}: click **Actions** under "{% octicon "law" aria-label="The law icon" %} Policies".
+   * **In an organization or repository**: click **Actions** in the left sidebar{% if currentVersion == "free-pro-team@latest" %}, then click **Runners**{% endif %}.
```
Comment (lecoursen) [inline data/reusables/github-actions/self-hosted-runner-navigate-to-repo-org-enterprise.md:9]:
```suggestion
   * **In an organization or repository**: Click **Actions** in the left sidebar{% if currentVersion == "free-pro-team@latest" %}, then click **Runners**{% endif %}.
```

File: content/actions/hosting-your-own-runners/removing-self-hosted-runners.md [hunk 8]
```diff
@@ -51,7 +51,7 @@ To remove a self-hosted runner from an organization, you must be an organization
 
 {% data reusables.organizations.navigate-to-org %}
 {% data reusables.organizations.org_settings %}
-{% data reusables.repositories.settings-sidebar-actions %}
+{% data reusables.github-actions.settings-sidebar-actions-runners %}
 {% data reusables.github-actions.self-hosted-runner-removing-a-runner %}
 
 ### Removing a runner from an enterprise
```
Comment (lecoursen) [inline content/actions/hosting-your-own-runners/removing-self-hosted-runners.md:57]: I think we still have a missing step [here](https://docs-6161--patch-1.herokuapp.com/en/actions/hosting-your-own-runners/removing-self-hosted-runners#removing-a-runner-from-an-enterprise) to click the Runners tab.

Comment (martin389) [reply content/actions/hosting-your-own-runners/removing-self-hosted-runners.md:57]: Fixed, thank you!

**Review by lucascosti — DISMISSED** [review]
Just a few comments:

* Non interacterable UI elements should be in "quotes" not _italics_.
* IMO, we should track adding GHES versioning for these new conditions somewhere (e.g. a new issue). I can see us forgetting about this when it comes time for the next release 🙂

**Comment by martin389** [comment]
> * Non interacterable UI elements should be in "quotes" not _italics_.
> * IMO, we should track adding GHES versioning for these new conditions somewhere (e.g. a new issue). I can see us forgetting about this when it comes time for the next release 🙂

Thanks @lucascosti - I've added the the UI element fixes, and raised an internal issue that will check if these changes apply to a future product release.

**Review by lucascosti — APPROVED** [review]
⚡

**Pull request merged**